Rubberduck.VBEEditor/Events/VbeEvents.cs

```diff
@@ -53,6 +53,11 @@ private VBEEvents(IVBE vbe)
             
             _projects = vbe.VBProjects;
 
+            if (_projects == null)
```

Comment:
This is deterministically never null - `VBProjects` is returned wrapped in a SCW. It should be `if(_projects.IsWrappingNullReference)`

Reply:
I added that for the unit tests' sake since that was returning `null` as I didn't set it up and I got worried that there might be a weird edge case where the `IVBE` might decide to dispense a `null` for the `VBProjects` collection so figured an extra guard wouldn't hurt.

Reply:
In that case, wouldn't you want the test to fail?